### ISSUE TITLE
Any way to send a message to a channel unprompted?

### ISSUE THREAD

**Issue by avehlies** [rmmh/skybot#105]
I'd like to make a plugin that scrapes an RSS feed every 15-30 minutes and stores any new news items in a database. I'd like any new news items to be sent to the channel, but I can't seem to find any way to send a message without being prompted (by a command). I tried poking around with the say() and message() methods in the Input class, but couldn't get anywhere with those. Does anyone have any ideas?

**Comment by elitan** [comment]
I am working on a fork of skybot, a bot that will be able to be called from a client (your rss script) and output any given message to any given channel in any given network. All done asynchronously.

Have a look: https://github.com/elitan/skybot

--------

I dont know, should I trie to make a pull request of my code to this repo?

**Comment by lukeroge** [comment]
Yeah, it was renamed at some point in my bot. Not entirely sure why, but theres no really any direct backwards compatibility between skybot and modern cloudbot due to the differing python versions anyway!

**Comment by gtwy** [comment]
Elitan, did you ever implement the timer? I was trying to look through your code to see how it was done. I am in need of this feature.

**Comment by Red-M** [comment]
Check PR #171

**Comment by Red-M** [comment]
I've provided #171 to provide this requested hook.

**Comment by gtwy** [comment]
@Red-M Your request hook did work. Thank you!
